tsconfig.json

```diff
@@ -17,7 +17,9 @@
         "noImplicitAny": true,
         "strictFunctionTypes": true,
         "strictNullChecks": true,
-        "noImplicitThis": true
+        "noImplicitThis": true,
+        "esModuleInterop": true,
+        "skipLibCheck": true
```

Comment:
Without setting this, we end up with type errors in the [K8s client](https://github.com/kubernetes-client/javascript/issues/864), the [glob types](https://github.com/isaacs/node-glob/issues/529), and [shelljs](https://github.com/shelljs/shelljs/issues/1148).